src/index.ts

```diff
@@ -21,24 +30,25 @@ const server = new ApolloServer({
 const app = express();
 
 app.use(cors());
+app.use(xss());
+app.use(helmet());
+app.use(bodyParser.json());
 
-// Forces calls to have a valid token in the Authorization header
-app.use(
-  jwt({
-    secret: jwksRsa.expressJwtSecret({
-      cache: true,
-      rateLimit: true,
-      jwksRequestsPerMinute: 5,
-      jwksUri: `https://${process.env.TENANT}.auth0.com/.well-known/jwks.json`,
-    }),
-
-    // audience: 'not-specified', (ex: helpers-consumer-test)
-    issuer: `https://${process.env.TENANT}.auth0.com/`,
-    algorithms: ["RS256"],
-  })
-);
+app.use(passport.initialize());
+passport.use('jwt', jwtStrategy);
+
+const router = routes(express.Router());
+app.use(router)
+
+app.use('/api/graphql', auth());
```

Comment:
This needs to be disabled based on admin access

Reply:
Let's leave TODOs for security concerns

Reply:
Done